modules/crypt.cpp

```diff
@@ -65,9 +189,17 @@ class CCryptMod : public CModule {
         AddCommand("ListKeys", static_cast<CModCommand::ModCmdFunc>(
                                    &CCryptMod::OnListKeysCommand),
                    "", "List all keys");
+        AddCommand("KeyX", static_cast<CModCommand::ModCmdFunc>(
```

Comment:
Key eXchange, for some reason I feel like this abbreviation is common in crypto-related stuff, but I agree that it is not self-explanatory.